example/src/MultipleMapsScreen.tsx

```diff
@@ -40,6 +40,7 @@ import {
 import usePermissions from './checkPermissions';
 import OverlayModal from './overlayModal';
 import { NavigationView } from '../../src/navigation/navigationView/navigationView';
+import MapView from '../../src/maps/mapView/mapView';
```

Comment:
both navigationView and Mapview should be imported from the @googlemaps/react-native-navigation-sdk

make sure that new mapView is properly exported in the index.ts files in plugin code
seems like index.ts does not have changes, so it is missing